Libraries/LibLine/Editor.cpp

```diff
@@ -222,8 +222,14 @@ String Editor::get_line(const String& prompt)
         }
 
         Utf8View input_view { StringView { m_incomplete_data.data(), valid_bytes } };
+        size_t index_in_view = 0;
```

Comment:
And the index is — the index, counting in codepoints (not bytes)?

Reply:
well yes, it's `index_in_view`, you're thinking of a better name?

Reply:
`number_of_codepoints`? idk, I'm not familiar with LibLine internals, just bringing up that `Utf8View` doesn't actually support indexing, so "index in view" doesn't tell me much

Reply:
does `consumed_codepoints` sound ok?

Reply:
Yeah